tsconfig.json

```diff
@@ -7,7 +7,18 @@
     "emitDeclarationOnly": true,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "paths": {
+      "display-fetch_stream": ["./src/display/fetch_stream"],
+      "display-l10n_utils": ["./web/l10n_utils"],
+      "display-network": ["./src/display/network"],
+      "display-node_stream": ["./src/display/node_stream"],
+      "display-node_utils": ["./src/display/node_utils"],
+      "display-svg": ["./src/display/svg"],
+      "web-com": ["./web/genericcom"],
+      "web-print_service": ["./web/pdf_print_service"],
```

Comment:
These two should not be necessary, since they are only used in the default viewer and not actually in the "viewer components".

Reply:
I removed them